src/popup/components/AccountDetailsBase.vue

```diff
@@ -157,16 +160,22 @@ export default defineComponent({
       },
     );
 
+    watch(
+      () => props.ionicLifecycleStatus,
+      (value) => {
+        if (value === 'didEnter') {
```

Comment:
What if the value is different? Maybe we need to unobserve?

Reply:
`ResizeObservers` are supposed to disconnect when the element is removed. But for clarity we can call `disconnect()`